HGustavs/LenaSYS
# changed press duration for the code example list to show when press on forward or backward button

One can press down the forward or backward button in the global navigation to show a list of the previous and next code examples. To make the list show quicker the press duration was changed from 1000ms to 300ms.

Testing:

1. Go to course Webbprogrammering
2. Klick one of the code examples
3. Press the forward or backward button in the global navigation bar 
![image](https://user-images.githubusercontent.com/81631818/168004706-5f542ce0-24e1-4ee4-aecc-2468f7dfdcae.png)

Before it should take lolnger time for the list to show whereas the displaying of list should feel more responsive on this branch. The change is minimal though.
![image](https://user-images.githubusercontent.com/81631818/168004886-35d14c27-e76d-40fe-9100-c4b270c1a2ea.png)
